src/Components/Books.js

```diff
@@ -16,24 +25,54 @@ const Books = () => {
   return (
     <div className="books-container">
       <div className="books">
-        <ul>
+        <ul className="list-books">
           {books.isLoading ? (
-            <>
-              <MoonLoader color="black" />
-            </>
+            <div className="loader">
+              <MoonLoader color="black" width="30px" />
```

Comment:
- [ ] You did a very great job to ensure that we have a design that follows the template design. However, I suggest avoiding using the inline styling but still, you can consider using react inline styling that I can see you used above. This is to make sure that our code is organized and maintainable ☺️.